website/docs/d/resourcegroupstaggingapi_resources.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Resource Groups Tagging API"
+layout: "aws"
+page_title: "AWS: aws_resourcegroupstaggingapi_resources"
+description: |-
+  Provides details about resource tagging.
+---
+
+# Data Source: aws_resourcegroupstaggingapi_resources
+
+Provides details about resource tagging.
+
+## Example Usage
+
+### Get All Resource Tag Mappings
+
+```hcl
+data "aws_resourcegroupstaggingapi_resources" "test" {}
+```
+
+### Filter By Tag Key and Value
+
+```hcl
+data "aws_resourcegroupstaggingapi_resources" "test" {
+  tag_filters {
+    key    = "tag-key"
+    values = ["tag-value-1", "tag-value-2"]
+  }
+}
+```
+
+### Filter By Resource Type
+
+```hcl
+data "aws_resourcegroupstaggingapi_resources" "test" {
+  resource_type_filter = ["ec2:instance"]
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `exclude_compliant_resources` - (Optional) Specifies whether to exclude resources that are compliant with the tag policy. You can use this parameter only if the `include_compliance_details` argument is also set to `true`.
+* `include_compliance_details` - (Optional) Specifies whether to include details regarding the compliance with the effective tag policy.
+* `tag_filters` - (Optional) Specifies a list of Tag Filters (keys and values) to restrict the output to only those resources that have the specified tag and, if included, the specified value. See [Tag Filters](#tag-filters) below. Conflicts with `resource_arn_list`.
```

Comment:
I suggest that this attribute is renamed to `tag_filter` to match the `filter` attribute for EC2 (and other service) data sources.